### ISSUE TITLE
aria label

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/gsoc22_guided-tour#124]
why did you add aria label to the buttons in the tour?

the first rule of aria is not to use it

unless I am missing something there is no reason to add aria-label here and it should be removed

the only place it is valid is on the X for close

**Comment by obuisard** [comment]
You are right, I have been overzealous. Working on fixing it.

**Comment by brianteeman** [comment]
confirmed as fixed
